php_phongo.c

```diff
@@ -1269,13 +1318,6 @@ const mongoc_read_prefs_t* phongo_read_preference_from_zval(zval *zread_preferen
 
 	return NULL;
 } /* }}} */
-
-const php_phongo_query_t* phongo_query_from_zval(zval *zquery TSRMLS_DC) /* {{{ */
```

Comment:
No longer needed since `phongo_execute_query()` now takes the Query zval and runs it through `Z_QUERY_OBJ_P()`.